packages/core/src/components/tag-input/_tag-input.scss

```diff
@@ -114,7 +114,7 @@ $tag-input-icon-padding-large: ($pt-input-height-large - $pt-icon-size-large) /
     background-color: $input-background-color;
     box-shadow: input-transition-shadow($input-shadow-color-focus, true), $input-box-shadow-focus;
 
-    @each $intent, $color in $pt-intent-text-colors {
```

Comment:
fixes the colors for tag input active borders
![Screen Shot 2021-12-02 at 16 20 41](https://user-images.githubusercontent.com/5934779/144505065-0bbf928a-7675-4d02-b2b0-31354a2bd56a.png)